src/Idris/Imports.hs

```diff
@@ -94,15 +95,17 @@ findPkgIndex p = do let idx = pkgIndex p
 installedPackages :: IO [String]
 installedPackages = do
   idir <- getIdrisLibDir
-  filterM (goodDir idir) =<< dirContents idir
+  filterM (goodDir idir) =<< concat <$> (mapM dirContents (splitOn ":" idir))
```

Comment:
Doesn’t this break on Windows, as `:` is used to indicate a drive prefix? What separator is used on Windows for this kind of environment variable? Is there a standard function to do this?